src/ios/CDVCamera.m

```diff
@@ -525,8 +525,24 @@ - (void)resultForImage:(CDVPictureOptions*)options info:(NSDictionary*)info comp
 
 - (CDVPluginResult*)resultForVideo:(NSDictionary*)info
 {
-    NSString* moviePath = [[info objectForKey:UIImagePickerControllerMediaURL] absoluteString];
-    return [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsString:moviePath];
+    NSURL *url = [info objectForKey:UIImagePickerControllerMediaURL];
+
+    NSData *urlData = [NSData dataWithContentsOfURL:url];
+    NSArray   *paths = NSSearchPathForDirectoriesInDomains(NSDocumentDirectory, NSUserDomainMask, YES);
```

Comment:
This and the lines below have the variable name aligned, but the line above doesn't. Why the inconsistency?

Reply:
I forgot to format the code ( which is a ctrl + I  on xcode)